base_layer/core/src/mining/primitives.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019. The Tari Project
```

Comment:
This should be clearly marked as a test struct. 

It's currently duped from the transactions crate.

Long term, the best approach is to have a test_utils crate that can be brought in as a `dev_dependency` for integration tests. This will remove the several other bits of code that are duped across `core` and `transactions`.

Short term, I'd rather have this in the `test_utils` module of core

Reply:
I agree with the long term goal. 

Moved this to test_utils of core